tools/RAiDER/models/hrrr.py

```diff
@@ -134,152 +141,123 @@ def load_weather(self, *args, filename=None, **kwargs):
         self._lons = _lons
 
 
-    def _makeDataCubes(self, filename, out=None):
-        '''
-        Create a cube of data representing temperature and relative humidity
-        at specified pressure levels
-        '''
-        if out is None:
-            out = filename
-
-        # Get profile information from gdal
-        prof = rio_profile(str(filename))
-
-        # Now get bounds
-        S, N, W, E = self._ll_bounds
+class HRRRAK(WeatherModel):
+    def __init__(self):
+        # The HRRR-AK model has a few different parameters than HRRR-CONUS. 
+        # These will get used if a user requests a bounding box in Alaska
+        super().__init__()
+        self._classname = 'hrrrak'
+        self._dataset = 'hrrrak'
+        self._Name = "HRRR-AK"
+        self._time_res = TIME_RES['HRRR']
+        self._valid_range = (datetime.datetime(2018, 7, 13), "Present")
+        self._lag_time = datetime.timedelta(hours=3)
+        self._valid_bounds =  Polygon(((195, 40), (157, 55), (175, 70), (260, 77), (232, 52)))
+
+        # This projection information will never get used but I'm keeping it for reference
+        # for the HRRR-AK model. The projection information gets read directly from the 
+        # weather model file. 
+        # self._proj = CRS.from_string(
+        #     '+proj=stere +ellps=sphere +a=6371229.0 +b=6371229.0 +lat_0=90 +lon_0=225.0 ' +
```

Comment:
Earlier a statement is made it gets used for buffering in advance without need to download the mode. If that is the case should the projection also not be updated?

Reply:
Fixed this.